superset/views.py

```diff
@@ -2383,12 +2383,16 @@ def table_accessible(database, full_table_name, schema_name=None):
                 get_datasource_access_error_msg('{}'.format(rejected_tables)))
         session.commit()
 
+        select_as_cta = request.form.get('select_as_cta') == 'true'
+        if select_as_cta and mydb.force_ctas_schema:
+            schema = mydb.force_ctas_schema
```

Comment:
I am afraid it will break sql lab.
schema for this endpoint should be one selected in the drop down menu.